app/models/language.rb

```diff
@@ -2,44 +2,12 @@
 #
 # Table name: languages
 #
+#  created_at :datetime
 #  id         :integer          not null, primary key
+#  locale     :string
 #  name       :string
-#  code       :string
-#  created_at :datetime
 #  updated_at :datetime
 #
 
 class Language < ActiveRecord::Base
```

Comment:
Last word I had on this was that we were going to bypass this model for the vast majority of the translation code, but that we were still going to need it to resolve locales to language names for the purposes of, for instance, crafting dropdowns for locale selection and for some things on the admin side for assigning multiple translations for a field at once with friendly language names. As far as I could tell, rails provides no other way of getting the language name for a locale.